src/index.ts

```diff
@@ -59,10 +59,14 @@ export class Paperless {
 
   /**
    * List documents. This function replaces the returned next and previous links with simple page numbers which you can feed back into the fn.
-   * @param page: Page to be queried
+   * @param filter - Filter for the result
+   * @param ordering - Ordering of the result list
+   * @param page - Page to be queried
    */
-  public async getDocuments(page = 1): Promise<DocumentListParsed> {
-    const response = await this.instance.get<DocumentListResult>(`/documents/?page=${page}`);
+   public async getDocuments(filter: DocumentFilter, ordering: DocumentOrdering, page = 1): Promise<DocumentListParsed> {
```

Comment:
Please keep the `page` at first place and make the other two parameters optional. That way we don't have a breaking change.
```suggestion
public async getDocuments(page = 1, filter?: DocumentFilter, ordering?: DocumentOrdering): Promise<DocumentListParsed> {
```